### PR TITLE
release notes Azure CLI 2.0.27

### PR DESCRIPTION
**Pull request by derekbekoe**

### CODE REVIEW

**Comment by derekbekoe** [comment]
@sptramer can you review?
cc: @twitchax as FYI

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/derekbekoe/azure-docs-cli-python/blob/az-2.0.27/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-634) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-2017-03-09-profile&branch=pr-en-us-634) (azure-cli-2017-03-09-profile) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C2%5C13%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201802131847007548-634%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=8%2BdBvkhk%2B9Y5EVJaSCy6FllefbPgAeWWoQfKvUMcKAg%3D&st=2018-02-13T18%3A44%3A34Z&se=2018-03-16T18%3A49%3A34Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/derekbekoe/azure-docs-cli-python/blob/az-2.0.27/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-634) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-2017-03-09-profile&branch=pr-en-us-634) (azure-cli-2017-03-09-profile) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C2%5C13%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201802131936423343-634%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=hT9NfSbtm7Q9cNNO8zVXvG938jEmZaWhE6zAvAvjrAQ%3D&st=2018-02-13T19%3A34%3A20Z&se=2018-03-16T19%3A39%3A20Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-conceptual/release-notes-azure-cli.md [hunk 1]
```diff
@@ -15,6 +15,108 @@ ms.service: multiple
...
 # Azure CLI 2.0 release notes
 
+## February 13, 2018
+
+Version 2.0.27
+
+### Core
+
+* Changed authentication to key on both subscription ID and name on MSI login
+* Add events module in core for EVENT_INVOKER_PRE_CMD_TBL_TRUNCATE [X]
```
[the remark this answers is not in the document]
Comment (derekbekoe) [reply docs-ref-conceptual/release-notes-azure-cli.md]: removed.

File: docs-ref-conceptual/release-notes-azure-cli.md [hunk 2]
```diff
@@ -15,6 +15,108 @@ ms.service: multiple
...
+
+### Role
+
+* Added `--available-to-other-tenants` argument to `ad app update`
+
+### SQL
+
+* Added `sql server dns-alias` commands
+* Added `az sql db rename`
+* Support `--ids` argument for db, dw, server, elastic-pool, and server firewall-rule command [X]
```
[the remark this answers is not in the document]
Comment (derekbekoe) [reply docs-ref-conceptual/release-notes-azure-cli.md]: yes it was for all commands.

**Comment by derekbekoe** [comment]
@sptramer feedback addressed.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/derekbekoe/azure-docs-cli-python/blob/az-2.0.27/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-634) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-2017-03-09-profile&branch=pr-en-us-634) (azure-cli-2017-03-09-profile) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C2%5C13%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201802132247143629-634%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ykLslojOYtSIwUJl%2BfTUDdpPkV5OJh31UKTANiXbEzw%3D&st=2018-02-13T22%3A44%3A45Z&se=2018-03-16T22%3A49%3A45Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/derekbekoe/azure-docs-cli-python/blob/az-2.0.27/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-634) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-2017-03-09-profile&branch=pr-en-us-634) (azure-cli-2017-03-09-profile) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C2%5C13%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201802132254325207-634%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=MdaJ%2Bz%2BrZ2QhfNX8VskNK%2FigdWB1%2FE%2Bzs%2BXSztqv4JU%3D&st=2018-02-13T22%3A51%3A54Z&se=2018-03-16T22%3A56%3A54Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/derekbekoe/azure-docs-cli-python/blob/az-2.0.27/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-634) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-2017-03-09-profile&branch=pr-en-us-634) (azure-cli-2017-03-09-profile) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C2%5C13%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201802132301290835-634%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=M7ti0iF9fT4EaupwSDV8tLUD%2Be2ULECZyPvy3FgKmKc%3D&st=2018-02-13T22%3A58%3A51Z&se=2018-03-16T23%3A03%3A51Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**